src/control/cmd/dmg/system.go

```diff
@@ -0,0 +1,82 @@
+//
+// (C) Copyright 2019 Intel Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
+// The Government's rights to use, modify, reproduce, release, perform, display,
+// or disclose this software are subject to the terms of the Apache License as
+// provided in Contract No. 8F-30005.
+// Any reproduction of computer software, computer software documentation, or
+// portions thereof marked with this legend must also reproduce the markings.
+//
+
+package main
+
+import (
+	"fmt"
+
+	"github.com/pkg/errors"
+)
+
+// systemCmd is the struct representing the top-level system subcommand.
+type SystemCmd struct {
+	Query systemQueryCmd `command:"query" alias:"q" description:"Retrieve DAOS system membership"`
+	Stop  systemStopCmd  `command:"stop" alias:"s" description:"Perform controlled shutdown of DAOS system"`
+}
+
+// systemStopCmd is the struct representing the command to shutdown system.
+type systemStopCmd struct {
+	logCmd
+	connectedCmd
+}
+
+// Execute is run when systemStopCmd activates
+func (cmd *systemStopCmd) Execute(args []string) error {
+	msg := "SUCCEEDED: "
+
+	members, err := cmd.conns.SystemStop()
+	if err != nil {
+		msg = errors.WithMessagef(err, "FAILED").Error()
+	}
+	if len(members) > 0 {
+		msg += fmt.Sprintf(": %+v", members)
```

Comment:
This seems like it could be excessively verbose. For tiny clusters it might be fine, but for clusters with tens, hundreds, thousands of members, this won't scale well. Perhaps it'd be better to just log the number of members that succeeded?

Also, one pattern to think about with these kinds of commands is that we'll probably want to display more than a binary choice between "it 100% worked" and "at least one failed, so the entire thing looks like it failed".

e.g. 
SUCCEEDED: 62
FAILED: 2

or better, but we don't yet have hostlist support:
SUCCEEDED: fnord[00-23,25-55,57-63]
FAILED: fnord[24,56]

Reply:
FYI: https://jira.hpdd.intel.com/browse/DAOS-3407

Reply:
makes sense

Reply:
have adjusted to try to meet this requirement in some way to stick to the scope of the PR, the rest of the changes to respond with number of failed attempts will be dealt with in a separate PR (another subtask in the parent PR) https://jira.hpdd.intel.com/browse/DAOS-3411